pkg/v1/progress.go

```diff
@@ -0,0 +1,7 @@
+package v1
+
+// Update representation of download update
```

Comment:
This comment could be a little more helpful -- calling out tarball.WithProgress here would be great.

I think "download update" is a bit too specific, since this could really represent any kind of progress update, independent of the image source.

Reply:
Sure, we can make this godoc a little more user-friendly and rational.

Reply:
Done